app/views/base/order_manager.html.erb

```diff
@@ -1,7 +1,7 @@
 <div class="container">
 <ul class="breadcrumb">
-  <% if decorated_change_set_resource.try(:decorated_parent_resource).present? %>
-    <li><%= link_to decorated_change_set_resource.decorated_parent_resource.first_title, solr_document_path(id: "#{decorated_change_set_resource.decorated_parent_resource.id}") %></li>
+  <% if decorated_change_set_resource.try(:decorated_parent).present? %>
+    <li><%= link_to decorated_change_set_resource.decorated_parent.first_title, solr_document_path(id: "#{decorated_change_set_resource.decorated_parent.id}") %></li>
   <% end %>
```

Comment:
Do you need .to_s here like you have in the file_manager.html.erb? 
`solr_document_path(id: "#{decorated_change_set_resource.decorated_parent.id}"`

Reply:
oh never mind.